Marlin/src/MarlinCore.cpp

```diff
@@ -476,7 +476,7 @@ inline void manage_inactivity(const bool no_stepper_sleep=false) {
   #endif
 
   #if HAS_FREEZE_PIN
-    Stepper::frozen = !READ(FREEZE_PIN);
+    Stepper::frozen = READ(FREEZE_PIN) ^ DISABLED(INVERT_FREEZE_PIN);
```

Comment:
Is INVERT_FREEZE_PIN still present?

Reply:
Nope. This will need another patch.

Reply:
The PR is all messed up now.
FREEZE_STATE only enables pullup or pull down and does not invert the logic level.
The entire original purpose of PR has been lost.

To many busy bodies and a rouge main programmer.
It enough to drive me around the bend.